src/components/create-card.jsx

```diff
@@ -2,10 +2,64 @@ import React from 'react';
 
 
 class CreateCard extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      question: '',
+      answer: ''
+    }
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.reset = this.reset.bind(this);
+  }
+
+handleChange(event){
+const name = event.target.name;
+const value = event.target.value;
+this.setState({
+  [name]: value
+});
```

Comment:
Watch your indenting, proper indenting makes your code easier to read and debug.